### DISCUSSION TITLE
FR: Ability to pull content from one entry into another

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#16291] [Ideas]
It would be great if we were given the ability to pull content from one entry into another. This functionality would serve various cases:

- **Pull content from a "regular" entry into a nested entry**. This would certainly come in handy, since, for the time being, we can't move or clone an entry from a section into a matrix block.
- Define, for example, **a "templates" section,** from where we could pull content into various new entries.
- Any other case where we would need to manually copy the content of an entry into another.
- Partly cover, in a much simpler way, the functionality of my [other idea ](https://github.com/craftcms/cms/discussions/16264) of "linked" master entries, entry instances etc.

Thanks for considering! 🙂

**Comment by humandevs** [comment]
This would also cover our use cases for composable entries, and along with scoped / conditional variables would provide most of the functionality of competing and expensive Enterprise Content Management systems such as Contentful.

**Reply by thupsi** [reply]
I have scoped variables right now in a project. Not sure about the scalability, but it works for my needs. I define the variables with their default value in a section. Then, in my entries I have a matrix field that can override the default if needed. The variables are used inside curly brackets in the content and replaced with a replace filter in the template.
